docs/annotations.md

```diff
@@ -96,3 +97,89 @@ class MyControl extends Control
     }
 }
 ```
+
+## `@phpstan-latte-template`
+Allowed context: render call, method, class
+
+Value specifies what template or templates should be resolved and checked in given context.
+
+Multiple annotations can be used on single element or at different levels to check multiple tempaltes with same context.
+
+### Behaviour
+
+When this annotation is used on render call it will replace value passed as tempalte to render call.
+
+When annotation is used on method it will resolve given template in context of this method (alongside renders used in method body).
+
+When annotation is used on class it ats like it was added to all methods in class.
+
+### Annotation value
+
+Can use wildcard `*` to match multiple files.
+
+Can use placeholders:
+- `{dir}` = `__DIR__` (example `/app/dir/controls`)
+- `{file}` = `__FILE__` (example `/app/dir/controls/MyControl.php`)
+- `{fileName}` = `basename(__FILE__)` (example `MyControl.php`)
+- `{fileNameNoSuffix}` = file anme with suffix removed (example `MyControl`)
```

Comment:
we could naming like https://www.php.net/manual/en/function.pathinfo.php
filename, basename, extension